src/test/scala/cromwell/engine/db/slick/SlickDataAccessSpec.scala

```diff
@@ -682,26 +682,22 @@ class SlickDataAccessSpec extends FlatSpec with Matchers with ScalaFutures with
       val task = Task.empty
       val call1 = new Call(None, callFqn1, task, Set.empty[FullyQualifiedName], Map.empty, None)
       val shardIndex1 = Option(0)
-      val pid1 = Option(123)
-      val backendInfo1 = new LocalCallBackendInfo(pid1)
+      val pid1 = Option("123")
       val call2 = new Call(None, callFqn2, task, Set.empty[FullyQualifiedName], Map.empty, None)
       val shardIndex2 = Option(1)
-      val pid2 = Option(987)
-      val backendInfo2 = new LocalCallBackendInfo(pid2)
+      val pid2 = Option("987")
 
       (for {
         _ <- dataAccess.createWorkflow(workflowInfo, Nil, Nil, localBackend)
         _ <- dataAccess.insertCalls(workflowId, Seq(BackendCallKey(call1, shardIndex1)), localBackend)
         _ <- dataAccess.insertCalls(workflowId, Seq(BackendCallKey(call2, shardIndex2)), localBackend)
-        _ <- dataAccess.updateExecutionBackendInfo(workflowId, BackendCallKey(call1, shardIndex1), backendInfo1)
-        _ <- dataAccess.updateExecutionBackendInfo(workflowId, BackendCallKey(call2, shardIndex2), backendInfo2)
+        _ <- dataAccess.updateExecutionInfo(workflowId, BackendCallKey(call1, shardIndex1), "PID", pid1)
+        _ <- dataAccess.updateExecutionInfo(workflowId, BackendCallKey(call2, shardIndex2), "PID", pid2)
```

Comment:
backend info keys should be constants in their Backend or BackendType companion objects